libs/ui-lib/lib/common/components/ui/WizardFooter.tsx

```diff
@@ -95,6 +97,13 @@ export const WizardFooter: React.FC<WizardFooterProps> = ({
               </Button>
             </ActionListItem>
           )}
+          {onReset && cluster && (
+            <ActionListItem>
+              <Button variant={ButtonVariant.link} name="reset" onClick={onReset}>
```

Comment:
the btn should be disabled while submitting
```suggestion
              <Button variant={ButtonVariant.link} name="reset" onClick={onReset} isDisabled={isSubmitting}>
```